saas_demo/security/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_saas_demo,access_saas_demo,model_saas_demo,base.group_user,1,0,0,0
-access_saas_demo_repo,access_saas_demo_repo,model_saas_demo_repo,base.group_user,1,0,0,0
+access_saas_demo,access_saas_demo,model_saas_demo,base.group_user,1,1,1,0
+access_saas_demo_repo,access_saas_demo_repo,model_saas_demo_repo,base.group_user,1,1,1,0
```

Comment:
You shall not grant access to all internal users. Use ``saas.group_user`` instead